tests/bluetooth/tester/src/btp_mesh.c

```diff
@@ -12,16 +12,22 @@
 #include <zephyr/bluetooth/testing.h>
 #include <zephyr/bluetooth/mesh/cfg.h>
 #include <zephyr/sys/byteorder.h>
+#include <zephyr/settings/settings.h>
 #include <app_keys.h>
 #include <va.h>
+#include <sar_cfg_internal.h>
+#include <string.h>
 
 #include <zephyr/logging/log.h>
 #define LOG_MODULE_NAME bttester_mesh
 LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
 
 #include "btp/btp.h"
+#include "dfu_slot.h"
 
 #define CID_LOCAL 0x05F1
+#define COMPANY_ID_LF 0x05F1
+#define COMPANY_ID_NORDIC_SEMI 0x05F9
```

Comment:
Is it required to use Nordic's ID here? Does Zephyr community have its own ID?

Reply:
> Is it required to use Nordic's ID here?
I don't know tbh, have no idea why it is here. It is used in Health server test info. Do you suggest to remove it?

> Does Zephyr community have its own ID?
The one above (`COMPANY_ID_LF`)?